src/test/java/tests/BaseTest.java

```diff
@@ -4,9 +4,15 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
 
 public abstract class BaseTest {
```

Comment:
```suggestion
public class BaseTest {
```
Тебе с твоими тестами класс не нужно что бы был абстрактный (как дочитаешь поймешь)